hardhat.config.js

```diff
@@ -82,7 +82,7 @@ module.exports = {
 	},
 	paths: {
 		sources: "./contracts",
-		tests: "./tests-js",
+		tests: "./tests-js/new",
```

Comment:
Why the test folder is changed to `/new`, I can only see one test written in that in your branch, please correct me if I am wrong.

Reply:
This is not completed one, for testing conveninece, I updated like this, this will be resolved after the completion of test